lib/json_rails_logger/json_formatter.rb

```diff
@@ -57,7 +57,7 @@ class JsonFormatter < ::Logger::Formatter # rubocop:disable Metrics/ClassLength
     REQUEST_METHODS = %w[GET POST PUT DELETE PATCH].freeze
 
     # rubocop:disable Metrics/MethodLength
-    def call(severity, timestamp, _progname, raw_msg) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity
+    def call(severity, timestamp, _progname, raw_msg) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
This feels like a sign to do some refactoring. Maybe a new class dedicated to generating these messages is required.

Reply:
I completely agree @joescottdave, I'll add a ticket to the repo for refactoring!